freud/order.pyx

```diff
@@ -412,13 +413,18 @@ cdef class Translational(_PairCompute):
         **2D:** :class:`freud.order.Translational` is only defined for 2D
         systems. The points must be passed in as :code:`[x, y, 0]`.
 
+    .. note::
```

Comment:
Might need a newline below this for consistency with the rest of our docs -- please check.

Reply:
I looked at the example directly above this one, as well as files like msd and pmft, and I don't see such a newline anywhere, so this seems fine.

Reply:
Also built and double-checked just to be sure.